lib/course_planner_web/controllers/event_controller.ex

```diff
@@ -0,0 +1,76 @@
+defmodule CoursePlannerWeb.EventController do
+  @moduledoc false
+  use CoursePlannerWeb, :controller
+
+  alias CoursePlanner.Events
+  alias CoursePlanner.Events.Event
+
+  import Canary.Plugs
+  plug :authorize_controller
+
+  def index(conn, _params) do
+    events = Events.all()
```

Comment:
can you sort this in descending order, so the latest event be on top?
this way it's more user friendly

Reply:
plus maybe we actually should do what is already is in task index page. to have past event, and upcoming separated ? what do you think ?

Reply:
Probably.
I'll leave both to a different PR since I'll have to undo/redo/do differently the sorting.

Reply:
then lets have a card for it so we don't forget